### DISCUSSION TITLE
data-*="" kalıbı için yeni ad önerileri

### DISCUSSION THREAD

**Discussion by yankikucuk** [data-deveb/deveb.css#4] [Soru-Yanıt]
"data-kçk" gibi bir tanımlama yapmak gerçekten güzel Türkçemize yakışmıyor. Bunun yerine "veri-kçk" gibi, çıtayı bir kademe daha yukarı taşımak bize yakışacak olandır diye düşünüyorum. "veri" yerine başka öneriler olursa yine bu tartışma başlığının altına yazabilir.

Şimdiden yaşasın Türkçe yazılım!

**Comment by volkantash** [comment]
"data-" ayarlarını özelleştirilebilir kıldık. Önden tanımlı değeri "veri-" olarak ayarlayabiliriz. Dileyen ayarlardan bu adlandırmaları değiştirebiliyor. 🤗

**Comment by volkantash** [comment]
Şu ara aşağıdaki gibi bir ad düzeni sunuyorum. Öneriler ve tepkiler altında değişme, özgerme gerçekleşebilir.

data-kçk="" yerine .küçük=""
data-ort="" yerine .orta=""

**Reply by yankikucuk** [reply]
volkan cidden suyunu çıkarttın üzerine mum falan dikme aşamasındasın. özgerme ne ama ya. soru değil. cevap verme lütfen, isyanımı yansıtmak istedim sadece.

**Reply by volkantash** [reply]
Sevgili Türkmen kardaşlar. Sizi @yankikucuk ile bugün tanıştırmak istiyorum. Dilimizin enginliğinde biraz gezerek daha yakından tanışmanızı dilerim.

http://www.enedilim.com/sozluk/soz/%C3%B6zgermek
